client/src/components/TransferFunds.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+
+import { NumberInput } from 'plaid-threads/NumberInput';
+import { Button } from 'plaid-threads/Button';
+import { currencyFilter } from '../util';
+
+interface Props {}
+const TransferFunds: React.FC<Props> = (props: Props) => {
+  const [transferAmount, setTransferAmount] = useState('');
+  const handleSubmit = (e: any) => {
+    e.preventDefault();
+    // checkAmount(); logic to check if amount is less than balance/get available balance
```

Comment:
this is just an informative comment to let users know they should implement this function right?

I wonder if we should be more clear here - `You should implement this logic to check if...`

Reply:
this is a placeholder for me to add logic later...